<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LastFMSettingsPage</class>
 <widget class="QWidget" name="LastFMSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Last.fm</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="LoginStateWidget" name="login_state" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="login_container" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>28</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="login">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Clicking the Login button will open a web browser.  You should return to Clementine after you have logged in.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Preferences</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="scrobble">
        <property name="text">
         <string>Scrobble tracks that I listen to</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="love_ban_">
        <property name="text">
         <string>Show the &quot;love&quot; button</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="scrobble_button">
        <property name="text">
         <string>Show the scrobble button in the main window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="prefer_albumartist">
        <property name="text">
         <string>Use Album Artist tag when available</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LoginStateWidget</class>
   <extends>QWidget</extends>
   <header>widgets/loginstatewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrobble</tabstop>
  <tabstop>love_ban_</tabstop>
  <tabstop>scrobble_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
